yaksh/templates/yaksh/course_detail.html

```diff
@@ -41,6 +41,19 @@
         <a href="{{URL_ROOT}}/exam/manage/send_mail/{{ course.id }}/">
         Send Mail</a>
         </li>
+        <li>
+            <form id="upload_users" action="{{ URL_ROOT }}/exam/manage/upload_users/{{course.id}}/"
```

Comment:
Internally its form, visually for end user its just an upload button.
I agree that it should have been better UI, but the current course details page is such that it looks hanging on the page with other elements.

Reply:
I have currently added form on the top of the page, removed from sidebar, toggle thing I am not doing currently because the upload details are returned to the main course detail page. So it will appear as if upload and upload details are on different page.